lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb

```diff
@@ -50,21 +56,27 @@ def target_definition
             target.target_definition
           end
 
-          # Ensure that vendored static frameworks and libraries are not linked
-          # twice to the aggregate target, which shares the xcconfig of the user
-          # target.
-          #
           def custom_build_settings
             settings = {
               'CODE_SIGN_IDENTITY[sdk=appletvos*]' => '',
               'CODE_SIGN_IDENTITY[sdk=iphoneos*]'  => '',
               'CODE_SIGN_IDENTITY[sdk=watchos*]'   => '',
+              'PRODUCT_BUNDLE_IDENTIFIER'          => 'org.cocoapods.${PRODUCT_NAME:rfc1034identifier}',
+              'SKIP_INSTALL'                       => 'YES',
               'MACH_O_TYPE'                        => 'staticlib',
               'OTHER_LDFLAGS'                      => '',
               'OTHER_LIBTOOLFLAGS'                 => '',
```

Comment:
these are left as build settings due to some checks we have for user projects that override these